src/rvoice/fluid_rvoice_event.c

```diff
@@ -229,7 +229,7 @@ new_fluid_rvoice_eventhandler(int is_threadsafe, int queuesize,
    * that too many events are dispatched too early, causing incorrectly timed audio
    */
   eventhandler->is_threadsafe = TRUE;
-  eventhandler->queue_stored = 0;
+  fluid_atomic_int_set(&eventhandler->queue_stored, 0);
```

Comment:
We are in a constructor function. I dont see why this is necessary.

Reply:
atomic types needs to be opaque types for one to use a different implementation